app/models/post_presenter.rb

```diff
@@ -0,0 +1,6 @@
+module PostPresenter
+
+  def format_time
+    return "posted at #{self.created_at.strftime('%H:%M')} on #{self.created_at.strftime('%d/%m/%Y')}"
+  end
+end
```

Comment:
The "posted at etc" is purely view based, and should appear in the view template if at all possible.